tools/pot/tests/test_model_params.py

```diff
@@ -0,0 +1,129 @@
+    def __init__(self, config):
+        self._samples = self._generate_synthetic_data(config)
+
+    def __len__(self):
+        return len(self._samples)
+
+    def __getitem__(self, index):
+        return self._samples[index], index
+
+    def _generate_synthetic_data(self, config):
+        samples = []
+        np.random.seed(config.seed)
+        for _ in range(config.num_samples):
+            sample = np.random.normal(size=config.input_shape)
+            samples.append(sample)
+
+        return samples
+
+
+TEST_TRANSFORMERS_MODELS = [
+    (
+        'transformer_example',
+        'pytorch',
+        'DefaultQuantization',
+        {
+            'preset': 'performance',
+            'stat_subset_size': 10,
+            'threshold': 1000,
+            'target_device': 'CPU'
+        },
+        'transformer_example_fbc'
+    ),
+    (
+        'transformer_example',
+        'pytorch',
+        'DefaultQuantization',
+        {
+            'preset': 'performance',
+            'stat_subset_size': 10,
+            'threshold': 1000,
+            'target_device': 'CPU',
+            'use_fast_bias': False
+        },
+        'transformer_example_hbc'
+    )
+]
+
+
+@pytest.fixture(scope='module', params=TEST_TRANSFORMERS_MODELS,
+                ids=['{}_{}_{}'.format(*m) for m in TEST_TRANSFORMERS_MODELS])
+def _params(request):
+    return request.param
+
+
+def test_transformer_biases_after_correction(_params, tmp_path, models):
+    model_name, framework, algorithm, algorithm_params, test_name = _params
+
+    references_dir = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                  './data/reference_biases')
+    reference_path = os.path.join(references_dir, f'{test_name}.json')
+    reference_exists = os.path.isfile(reference_path)
+    local_path = os.path.join(tmp_path, f'{test_name}.json')
+
+    model = models.get(model_name, framework, tmp_path)
+    algorithm_config = Dict({
+        'algorithms': [{
+            'name': algorithm,
+            'params': algorithm_params
+        }]
+    })
+    engine_config = {'device': 'CPU'}
+    config = merge_configs(model.model_params, engine_config, algorithm_config)
+
+    data_loader_config = Dict({
+        'num_samples': algorithm_params['stat_subset_size'],
+        'seed': 0,
+        'input_shape': (1, 128)
+    })
+    data_loader = SyntheticDataLoader(data_loader_config)
+
+    engine = IEEngine(config.engine, data_loader=data_loader)
+    pipeline = create_pipeline(config.compression.algorithms, engine)
+    model = load_model(config.model)
+    compressed_model = pipeline.run(model)
+
+    values = {}
+
+    nodes_with_bias = get_nodes_by_type(
+        compressed_model, [op['type'] for op in OPERATIONS_WITH_BIAS])
+    for node_with_bias in nodes_with_bias:
+        bias = get_bias_for_node(node_with_bias)
+        if bias is not None:
+            values[node_with_bias.name] = get_node_value(bias)
```

Comment:
`get_nodes_by_type` will recursively find nodes in the model so below should be `fullname` 
```suggestion
            values[node_with_bias.fullname] = get_node_value(bias)
```